site/content/docs/4.3/components/popovers.md

```diff
@@ -299,7 +299,7 @@ Options for individual popovers can alternatively be specified through the use o
 
 #### show
 
-Reveals an element's popover. **Returns to the caller before the popover has actually been shown** (i.e. before the `shown.bs.popover` event occurs). This is considered a "manual" triggering of the popover. Popovers whose both title and content are zero-length are never displayed.
+Reveals an element's popover. **Returns to the caller before the popover has actually been shown** (i.e. before the `shown.bs.popover` event occurs). This is considered a "manual" triggering of the popover. Popovers whose title and content both are zero-length are never displayed.
```

Comment:
maybe even "are both" instead of "both are"?

Reply:
"are both" ???

Popovers whose title and content **are both** zero-length are never displayed.

Reply:
Popovers whose title and content **both are** zero-length are never displayed.

Reply:
question of style, but "are both" flows better, i'd say. both ways are correct.

Reply:
Hmm.. Ok :)